portality/templates/doaj/article.html

```diff
@@ -61,16 +63,21 @@
     <div class="span4">
         <div class="box with-borders" style="height: 120px">
             {% if article.bibjson().journal_title %}
+                {% if article.bibjson().get_identifiers(idtype='pissn') or article.bibjson().get_identifiers(idtype='eissn') %}
+                    <p><strong>Journal</strong>:&nbsp;
+                    <a href={{ "/toc/" + article.bibjson().get_identifiers(idtype='pissn')[0] or article.bibjson().get_identifiers(idtype='eissn')[0]}}>{{ article.bibjson().journal_title }}</a></p>
```

Comment:
I was asking since I wasn't sure it even looked right, but now have tested it and it does work! Fantastic Jinja find.